notes/error.md

```diff
@@ -1,349 +1,444 @@
+
+
+$$
+\hat{x} = x + \Delta x
+$$
+
+Given this problem setup we can define the absolute error as:
+
+$$
+\begin{equation}
+\text{Absolute Error} = |x - \hat{x}|
+\end{equation}
+.$$
+
+This tells us how close our approximate result is to the actual answer.
+However, absolute error can become an unsatisfactory and
+misleading representation of the error depending on the magnitude of $$x$$.
+
+<table class="table">
+<colgroup>
+<col width="50%" />
+<col width="50%" />
+</colgroup>
+<thead>
+<tr class="header">
+<th style="color: black;">Case 1</th>
+<th style="color: black;">Case 2</th>
+</tr>
+</thead>
+<tbody>
+<tr class="odd">
+<td><span class="math inline">\(x = 0.1\)</span>, <span class="math inline">\(\hat{x} = 0.2\)</span></td>
+<td><span class="math inline">\(x = 100.0\)</span>, <span class="math inline">\(\hat{x} = 100.1\)</span></td>
+</tr>
+<tr class="even">
+<td><span class="math inline">\(\mid x - \hat{x} \mid = 0.1\)</span></td>
+<td><span class="math inline">\(\mid x - \hat{x} \mid = 0.1\)</span></td>
+</tr>
+</tbody>
+</table>
+
+
+In both of these cases, the absolute error is the same, 0.1.
+However, we would intuitively consider case 2 more accurate than case 1
+since our approximation is double the true value in case 1.
+Because of this, we define the relative error, which will be an error estimate
+independent of the magnitude. To obtain this we simply divide the absolute
+error by the absolute value of the true value.
+
+$$
+\begin{equation}
+\text{Relative Error} = \frac{|x - \hat{x}|}{|x|}
+\end{equation}
+$$
+
+If we consider the two cases again, we can see that the relative error will be much lower in the second case, since relative error is independent of magnitude.
+
+<table class="table">
+<colgroup>
+<col width="50%" />
+<col width="50%" />
+</colgroup>
+<thead>
+<tr class="header">
+<th style="color: black;">Case 1</th>
+<th style="color: black;">Case 2</th>
+</tr>
+</thead>
+<tbody>
+<tr class="odd">
+<td><span class="math inline">\(x = 0.1\)</span>, <span class="math inline">\(\hat{x} = 0.2\)</span></td>
+<td><span class="math inline">\(x = 100.0\)</span>, <span class="math inline">\(\hat{x} = 100.1\)</span></td>
+</tr>
+<tr class="even">
+<td><span class="math inline">\(\frac{\mid x - \hat{x} \mid}{\mid x \mid} = 1\)</span></td>
+<td><span class="math inline">\(\frac{\mid x - \hat{x} \mid}{\mid x \mid} = 10^{-3}\)</span></td>
+</tr>
+</tbody>
+</table>
+## Significant Digits/Figures
+
+**Significant figures** of a number are digits that carry meaningful information. They are digits beginning from the leftmost nonzero digit and ending with the rightmost "correct" digit, including final zeros that are exact. For example:
+
+*   The number 3.14159 has six significant digits.
+*   The number 0.00035 has two significant digits.
+*   The number 0.000350 has three significant digits.
+<br/>
+The number of significant figures tells us about how many positions of $$x$$ and $$\hat{x}$$ agree.

```

Comment:
do we want to differentiate between sigfigs of a *number* on its own and sigfigs of an *approximation result*?